pkgs/development/ocaml-modules/augeas/default.nix

```diff
@@ -0,0 +1,63 @@
+{
+  stdenv,
+  fetchurl,
+  fetchDebianPatch,
+  autoreconfHook,
+  makeWrapper,
+  pkg-config,
+  ocamlPackages,
```

Comment:
Usually, packages from `ocamlPackages` do not receive it as argument: please take `ocaml` and `findlib` separately as arguments (instead of the full `ocamlPackages`).